src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/DraftAdjudicationService.kt

```diff
@@ -287,8 +294,9 @@ class DraftAdjudicationService(
       it.dateTimeOfIncident = draftAdjudication.incidentDetails.dateTimeOfIncident
       it.handoverDeadline = draftAdjudication.incidentDetails.handoverDeadline
       it.isYouthOffender = draftAdjudication.isYouthOffender
-      it.incidentRoleCode = draftAdjudication.incidentRole.roleCode
-      it.incidentRoleAssociatedPrisonersNumber = draftAdjudication.incidentRole.associatedPrisonersNumber
+      it.incidentRoleCode = draftAdjudication.incidentRole!!.roleCode
```

Comment:
we can't allow this - it will lead to an NPE if users don't set the incident details.

Add a check to `completeDraftAdjudication` similar to that for `offenceDetails` and `incidentStatement`